tests/src/Extensions/DrupalExtension.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\nexteuropa\DrupalExtension\ServiceContainer.
+ */
+
+namespace Drupal\nexteuropa\Extensions;
+
+use Drupal\DrupalExtension\ServiceContainer\DrupalExtension as OriginalDrupalExtension;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+/**
+ * Class DrupalExtension.
+ *
+ * @package Drupal\nexteuropa\Extensions
+ */
+class DrupalExtension extends OriginalDrupalExtension {
+
+  /**
+   * Define extension definition.
+   *
+   * @inheritDoc
+   */
```

Comment:
I will change but do not use a Drupal reference in project that does not concern Drupal directly.